ondemand.osc.edu/app_config.yaml

```diff
@@ -0,0 +1,261 @@
+  # Type: String | null
+  # Default: null
+  uri: file:///etc/motd
+  ## motd.title
+  # !!Deprecated!!: Use dashboard.motd_title in the locale file instead
+  # Purpose: The title for the MOTD block
+  # Type: String | null
+  # Replaces: MOTD_TITLE
+  # Default: null
+  title: null
+## announcement_path
+# Purpose: The file system path to the announcements
+# Note: Could be an array; we support usage of an announcements.d we could pull from multiple paths
+# Type: String | Sequence<String>
+# Replaces: OOD_ANNOUNCEMENT_PATH
+# Default: null
+announcement_path: null
+## app_catalog_url
+# Purpose: The link to the external application catalog page
+# Type: String | null
+# Replaces: OOD_APP_CATALOG_URL
+# Default: null
+app_catalog_url: null
+## app_config_root
+# Purpose: The file system path to the default configuration files for specific apps
+# Note: 
+# Type: String | null
+# Replaces: OOD_APP_CONFIG_ROOT
+# Default:
+app_config_root: null
+## app_development
+# Purpose: Flag whether the developer view should be shown in the Dashboard
+# Note: Considers whether the development app directory exists in the configured path, or is a symlink; see also OOD_DEV_APPS_ROOT
+# Type: Boolean | null
+# Replaces: OOD_APP_DEVELOPMENT
+# Default: false
+app_development: null
+## APP_SHARING
+# Purpose: Flag whether the app sharing controls should be shown in the Dashboard
+# Type: Boolean | null
+# Replaces: OOD_APP_SHARING
+# Default: false
+app_sharing: false
+## BC_APP_CONFIG_ROOT
+# Purpose: The file system path to the default configuration files for Batch Connect applications
+# Type: String | null
+# Replaces: OOD_BC_APP_CONFIG_ROOT
+# Default: /etc/ood/config/apps
+bc_app_config_root: null
+## dashboard_support_email
+# Purpose: The email address for your site's user support staff
+# Type: String | null
+# Replaces: OOD_DASHBOARD_SUPPORT_EMAIL
+# Default:
+dashboard_support_email: oschelp@osc.edu
+## dev_apps_root
+# Purpose: Contains the path to the user-specific development directory
+# TODO: What sets this? Do we ever want to override it? Is it always user-specific?
+# Type: String | null
+# Replaces: OOD_DEV_APPS_ROOT
+# Default: /dev/null
+dev_apps_root: null
+## dev_ssh_host
+# Purpose: A web node, serving OnDemand, where developers can SSH to perform development work.
+# Type: String | null
+# Replaces: OOD_DEV_SSH_HOST
+# Default: null
+dev_ssh_host: ondemand-test.osc.edu
+## user_file_system_locations
+# Purpose: A list of file system locations that the user has access to
+# Type: Sequence<String> | null
+# Replaces: Custom Rails initializer
+# Default: null
+user_file_system_locations: &user_file_system_locations_ref
+  <% user_file_system_locations.each do |location| %>
+  - <%= location %>
+  <% end %>
+## links
+# Purpose: Define the OnDemand menu
+# Replaces:
+#   - DASHBOARD_2FA_URL
+#   - DASHBOARD_DEV_DOCS_URL
+#   - DASHBOARD_DOCS_URL
+#   - OOD_DASHBOARD_PASSWD_URL
+#   - OOD_DASHBOARD_SUPPORT_URL
+links:
+  Files: null
```

Comment:
If we are supporting ERB as suggested in https://github.com/OSC/ood_core/pull/154, should we add the ERB here to build the list of favorites?

Reply:
Oh. You are doing that with `user_file_system_locations` instead.

Reply:
Which  makes sense; we may display these links in multiple locations.